.circleci/continue_config.yml

```diff
@@ -187,36 +224,41 @@ workflows:
   ci:
     jobs:
       - ci-base:
-          name: ci-base-<< matrix.python-version >>
+          name: ci-base-<< matrix.python-version >>-<< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
+              extras: ["", "metrics"]
       - unit-test:
+          name: unit-test-<< matrix.python-version >>-<< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11" ]
+              extras: ["", "metrics"]
           requires:
-            - ci-base-<< matrix.python-version >>
+            - ci-base-<< matrix.python-version >>-<< matrix.extras >>
       - integration-test:
-          name: integration-test-<< matrix.pytest-group >>-<< matrix.python-version >>
+          name: integration-test-<< matrix.pytest-group >>-<< matrix.python-version >>-<< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.9" ]
               pytest-group: [ detection, fr, classification, workflow ]
+              extras: ["", "metrics"]
           enabled: true
           parallelism: 4
           requires:
-            - ci-base-<< matrix.python-version >>
+            - ci-base-<< matrix.python-version >>-<< matrix.extras >>
       - integration-test:
-          name: integration-test-misc-<< matrix.python-version >>
+          name: integration-test-misc-<< matrix.python-version >>-<< matrix.extras >>
           matrix:
             parameters:
               python-version: [ "3.9" ]
+              extras: ["", "metrics"]
```

Comment:
Nit: do we need both or one of them suffice?

Reply:
ah I see, `requires` structures change, but not the actual key